tests/postgres_tests/test_operations.py

```diff
@@ -173,16 +173,16 @@ def test_no_allow_migrate(self):
         self.assertEqual(len(captured_queries), 0)
 
     def test_allow_migrate(self):
-        operation = CreateExtension('uuid-ossp')
+        operation = CreateExtension('tablefunc')
```

Comment:
Please revert this unrelated change.

Reply:
please read this: https://github.com/django/django/pull/12950#issuecomment-632392309

Reply:
I understand why you bumped indexes in below assertions, but changing `'uuid-ossp'` is not related with this PR and should be moved to a separate commit/PR.

Reply:
I am confused.  That test is incorrect at the moment even without my PR.  `uuid-ossp` is already installed at this point into the database and a real `CREATE EXTENSION` is never run. That test passes only because of `IF NOT EXISTS`.

Reply:
Exactly, it's not **related**. That's why we should fix it separately in advance.

Reply:
I am sorry, but creating another PR and ticket and the whole shebang just seems like waaaay overkill if it was "fine" all these years...  But sure i'll change it back, feel free to push a fix for that.

Reply:
It's not an overkill. Mixing different kind of changes makes a project like Django extremely difficult to maintain.

I added new PR #12972 .

Reply:
I thought about it, but it has some value to test this explicitly.

Reply:
I think this test is not testing what it should be testing in it's current form.  See my answer to orf.

Reply:
right. so my test needs to insert a so far non-inserted extension.  what would you like me to do?